input.gms

```diff
@@ -198,6 +198,12 @@ $ondelim
 $include"./iResDemSub.csv"
 $offdelim
 ;
+table ICO2SeqData(allCy,CO2SEQELAST,YTIME)	       "Data for CO2 sequestration (1)" 
```

Comment:
why is the i in the parameter name capitalized?

Reply:
Fixed